dnf/package.py

```diff
@@ -225,9 +225,9 @@ def schemes_filter(url_list):
                 if schemes:
                     s = dnf.pycomp.urlparse.urlparse(url)[0]
                     if s in schemes:
-                        return url + self.location
+                        return os.path.join(url, self.location)
```

Comment:
@ignatenkobrain I would like to release the patch on next Monday. Hope that the merge will be possible.